README.md

```diff
@@ -1 +1,285 @@
-# yoti-face-capture-react-native
\ No newline at end of file
+# Yoti Face Capture for React Native
+An easy to use face detection component for React Native from Yoti.
+Face detection is performed with the front facing camera, whose frames the library analyzes and then produces an optimised cropped image of the captured face.
+
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/getyoti/react-native-yoti-face-capture?label=latest%20release)](https://github.com/getyoti/react-native-yoti-face-capture/releases) [![Publish Release](https://github.com/getyoti/react-native-yoti-face-capture/workflows/Publish%20Release/badge.svg)](https://github.com/getyoti/react-native-yoti-face-capture/actions?query=workflow%3A%22Publish+Release%22)
+<br>
+[![Platform - Android](https://img.shields.io/badge/platform-Android-3ddc84.svg?style=flat&logo=android)](https://www.android.com)
+[![Platform - iOS](https://img.shields.io/badge/platform-iOS-000.svg?style=flat&logo=apple)](https://developer.apple.com/ios)
+
+The library leverages [Google ML Kit](https://firebase.google.com/docs/ml-kit/detect-faces) for the face detection.
```

Comment:
That is only true for Android, for iOS we use the Vision library https://developer.apple.com/documentation/vision